doc/tutorials/04-lattice_boltzmann/04-lattice_boltzmann_part1.ipynb

```diff
@@ -9,11 +9,12 @@
     "#### Before you start:\n",
```

Comment:
<p>Again here the equations don't get typeset correctly in the notebook? Or is this due to the HTML export we perform later?</p><p> </p><ul><li>add a note that thermalization is optional</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/3893/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>